modules/core/src/test/java/org/apache/ignite/testframework/junits/multijvm/IgniteNodeRunner.java

```diff
@@ -135,18 +146,69 @@ public static void storeToFile(IgniteConfiguration cfg, String fileName,
     }
 
     /**
+     * Stores {@link IgniteInClosure} to file as xml.
+     *
+     * @param clos IgniteClosure.
+     * @param cfgFileName A name of file where the configuration was stored.
```

Comment:
What do we store? IgniteInClousre or configuration?
I don't get it from javadoc